packages/admin-ui/src/app/_services_and_types/participants.service.ts

```diff
@@ -66,9 +138,9 @@ export class ParticipantsService {
 			changeLog: [],
 			netDebitCapChangeRequests: [],
 			netDebitCaps: [],
-			participantContacts:[],
+			participantContacts: [],
```

Comment:
Avoid reformatting big important files like this.

Reply:
The main reason is the different IDEs and the different indentations between devs. I'm using VSCode and did format with the default TS & JS Language Features with 4 tabs (previously, indent using 2 spaces with Prettier). Sorry about that I'll try to adjust it by tomorrow's meeting.